packages/functions/src/index.node.ts

```diff
@@ -15,9 +15,9 @@
  * limitations under the License.
  */
 import { registerFunctions } from './config';
-import nodeFetch from 'node-fetch';
+import { fetch } from 'undici';
```

Comment:
maybe we should rename this to undiciFetch as well? Not 100% sure if this will be safe from collisions in Node 18+ where fetch is in the global scope?

Reply:
Done.